### PR TITLE
[Feat] - Added New Docs

### PR DESCRIPTION
**Pull request by adithyaakrishna**
**Description:**

This PR adds new docs which were missing from the book's site.

- Related PR - https://github.com/WasmEdge/WasmEdge/pull/2727
- Also, fixes https://github.com/WasmEdge/docs/issues/133

### CODE REVIEW

**Comment by alabulei1** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

**Comment by adithyaakrishna** [comment]
Oops! My bad, there are a hell lotta broken links, will fix them up :)

File: docs/embed/use-case/mesh/dapr.md [hunk 1]
```diff
@@ -0,0 +1,263 @@
+---
```
Comment (alabulei1) [inline docs/embed/use-case/mesh/dapr.md:1]: This tutorial is out of state. We don't need to keep it.

File: docs/embed/use-case/reactr.md [hunk 2]
```diff
@@ -0,0 +1,365 @@
+---
```
Comment (alabulei1) [inline docs/embed/use-case/reactr.md:1]: This tutorial is out of state too.

File: docs/embed/use-case/mesh/eventmesh.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (alabulei1) [inline docs/embed/use-case/mesh/eventmesh.md:1]: Let's remove the part first. When the development is finished, we can add one.

File: docs/embed/use-case/serverless/secondstate.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (alabulei1) [inline docs/embed/use-case/serverless/secondstate.md:1]: This is out of date too.

**Comment by adithyaakrishna** [comment]
@alabulei1 Could you please review this PR?

File: docs/embed/use-case/serverless/_category_.json [hunk 5]
```diff
@@ -0,0 +1,8 @@
+{
+  "label": "Serviceless Platforms",
```
Comment (alabulei1) [inline docs/embed/use-case/serverless/_category_.json:2]: This is not the use case for embedding.

File: docs/embed/use-case/wasm-smart-devices.md [hunk 6]
```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 3
```
Comment (alabulei1) [inline docs/embed/use-case/wasm-smart-devices.md:2]: This is not the use case for embedding.

File: docs/embed/use-case/web-app.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
+---
```
Comment (alabulei1) [inline docs/embed/use-case/web-app.md:1]: This is not the use case for embedding. This doc tells the user how to create a Wasm image.

File: docs/embed/use-case/serverless/_category_.json (hunk 5 above)
Comment (alabulei1) [reply docs/embed/use-case/serverless/_category_.json:2]: You didn't solve this problem. serverless is not the use case for embedding WasmEdge.

File: docs/embed/use-case/serverless/aws.md [hunk 8]
```diff
@@ -0,0 +1,272 @@
+---
+sidebar_position: 1
```
Comment (alabulei1) [inline docs/embed/use-case/serverless/aws.md:2]: Same problem.

File: docs/embed/use-case/serverless/netlify.md [hunk 9]
```diff
@@ -0,0 +1,189 @@
+---
```
Comment (alabulei1) [inline docs/embed/use-case/serverless/netlify.md:1]: Same problem

File: docs/embed/use-case/serverless/tencent.md [hunk 10]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (alabulei1) [inline docs/embed/use-case/serverless/tencent.md:1]: Same problem

File: docs/embed/use-case/wasm-smart-devices.md (hunk 6 above)
Comment (alabulei1) [reply docs/embed/use-case/wasm-smart-devices.md:2]: You didn't solve the problem.

File: docs/embed/use-case/web-app.md (hunk 7 above)
Comment (alabulei1) [reply docs/embed/use-case/web-app.md:1]: You can't solve the problems without changing the content.

File: docs/embed/use-case/serverless/_category_.json (hunk 5 above)
Comment (alabulei1) [reply docs/embed/use-case/serverless/_category_.json:2]: I saw your changes below. We should remove this chapter.

File: docs/embed/use-case/serverless/aws.md (hunk 8 above)
Comment (alabulei1) [reply docs/embed/use-case/serverless/aws.md:2]: Please remove this chapter.

File: docs/embed/use-case/serverless/netlify.md (hunk 9 above)
Comment (alabulei1) [reply docs/embed/use-case/serverless/netlify.md:1]: Please remove this chapter.

File: docs/embed/use-case/serverless/tencent.md (hunk 10 above)
Comment (alabulei1) [reply docs/embed/use-case/serverless/tencent.md:1]: Please remove this chapter.

File: docs/embed/use-case/serverless/vercel.md [hunk 11]
```diff
@@ -0,0 +1,191 @@
+---
```
[the remark this answers is not in the document]
Comment (alabulei1) [reply docs/embed/use-case/serverless/vercel.md:1]: Please remove this chapter

File: docs/embed/use-case/wasm-smart-devices.md (hunk 6 above)
Comment (alabulei1) [reply docs/embed/use-case/wasm-smart-devices.md:2]: Please remove this chapter.

File: docs/embed/use-case/web-app.md (hunk 7 above)
Comment (alabulei1) [reply docs/embed/use-case/web-app.md:1]: Please remove this chapter.

File: docs/start/usage/web-app.md [hunk 12]
```diff
@@ -0,0 +1,101 @@
+---
```
Comment (alabulei1) [inline docs/start/usage/web-app.md:1]: Again, this is not the use case of WasmEdge. It tells users how to create a docker image for users. You can discard this chapter. We have the same file on https://github.com/second-state/wasmedge-containers-examples/blob/main/simple_wasi_app.md

**Comment by alabulei1** [comment]
Hi  @adithyaakrishna 

Please check out my comments. Thanks. After you removing several docs, this PR  can be merged.

**Comment by alabulei1** [comment]
Also there was a CI fail, please check it out. Thanks.

File: docs/embed/use-case/wasm-smart-devices.md (hunk 6 above)
Comment (adithyaakrishna) [reply docs/embed/use-case/wasm-smart-devices.md:2]: @alabulei1 I have put it under overview/start itself. Had missed pushing one change, Just did it below

File: docs/start/usage/web-app.md (hunk 12 above)
Comment (adithyaakrishna) [reply docs/start/usage/web-app.md:1]: Gotcha

**Review by alabulei1 — APPROVED** [review]

**Pull request merged**